src/status_im2/contexts/chat/camera/view.cljs

```diff
@@ -43,59 +47,94 @@
                  (camera-kit/capture @camera-ref #(reset! uri %)))
      :style    style/inner-circle}]])
 
+(defn retake
+  [flash uri]
+  (let [current-flash @flash]
+    (when platform/android?
+      (reset! flash false) ; On Android, setting flash needs to be delayed until camera has initialized
+      (js/setTimeout #(reset! flash current-flash) 300))
+    (reset! uri nil)))
```

Comment:
Heads up: We probably need to disable Flash if the app goes background because of this same rendering issue.

Ref: https://github.com/status-im/status-mobile/pull/16558#issuecomment-1632597168

Reply:
I think this is a little bit different. We are not toggling the flash always on like in that QR screen. Going to background did not cause any problem.